.github/workflows/label-tweet.yml

```diff
@@ -0,0 +1,31 @@
+name: 'Tweet when Labeled'
+
+on:
+  issues:
+    types: labeled
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  action:
+    if: ${{ github.event.label.name == 'issue/tweet' }}
+    runs-on: ubuntu-latest
+    steps:
+        - name: Check if organization member
```

Comment:
@gopivaibhav more than this, will you try using this - action https://github.com/tspascoal/get-user-teams-membership - to assess whether the user is a member of @meshery/maintainers?

Reply:
Oh, okay. I will change it. Also, What am I supposed to tweet?
Is the content which I have written now looking good? Or do I have to change it to something else?

Reply:
I missed your message here. Let's try this:

```
Help wanted: "<issue title>" - <issue link> #opensource
```

Reply:
Added the required tweet body for the tweet